docs/src/main/sphinx/connector/optimize.fragment

```diff
@@ -16,7 +16,7 @@ The following statement merges files in a table that are
 under 10 megabytes in size:
```

Comment:
Sorry didn't pay close enough attention at first. @dominikzalewski please proofread the changed files for other 10mb references and update them too.

Reply:
Done